packages/babel-helpers/src/helpers.js

```diff
@@ -859,10 +859,14 @@ helpers.arrayWithHoles = helper("7.0.0-beta.0")`
 `;
 
 helpers.iterableToArray = helper("7.0.0-beta.0")`
+
   export default function _iterableToArray(iter) {
     if (
-      Symbol.iterator in Object(iter) ||
-      Object.prototype.toString.call(iter) === "[object Arguments]"
+      Array.isArray(iter)
+      || typeof iter === 'string'
+      || (typeof Symbol === 'function' && Symbol.iterator in Object(iter))
+      || (iter && 'length' in iter)
+      || Object.prototype.toString.call(iter) === "[object Arguments]"
     ) return Array.from(iter);
```

Comment:
So I encountered the [originating problem](https://github.com/babel/babel/issues/7597) on IE11 in the JS library I maintain as well, because we can't use `babel-polyfill` either for general compatibility. Wasn't sure how long this PR would take to go through so I tried seeing what would happen if we just polyfill `Symbol`, and that surfaced up this line as another issue: `Array.from` is also not supported in IE11.

So if you're encountering the `Symbol` problem in IE11, it's probably because you aren't using polyfills, which means this will likely be an issue as well. Not sure if that's tangential enough to be in a separate ticket entirely though.

Reply:
using es6-shim, Array.from works but symbols don’t, leading to this issue.

It’s reasonable to require shimming Array.from; but not to require shamming symbols.